template/util.template

```diff
@@ -7,7 +7,10 @@
 const util = require('util');
 const path = require('path');
 const fs = require('fs-extra');
-const process = require('process');
+let subjects = new Map();
+let aspects = new Map();
+let samples = new Map();
```

Comment:
If you're going with the global map approach, you also need to reset the maps each time processNode is called, otherwise the result will include entries from previous calls.